air/tests/test_module/negative_tests/preparation_step.rs

```diff
@@ -74,3 +75,45 @@ fn invalid_data_with_versions() {
     };
     assert!(check_error(&result, expected_error));
 }
+
+#[test]
+fn invalid_callresults() {
+    let air = r#"(null)"#.to_string();
+    let client_peer_id = "some_peer_id".to_string();
+    let prev_data = InterpreterData {
+        versions: Versions::new(semver::Version::new(1, 1, 1)),
+        trace: <_>::default(),
+        global_streams: <_>::default(),
+        restricted_streams: <_>::default(),
+        last_call_request_id: 0,
+        cid_info: <_>::default(),
+    };
+    let prev_data: Vec<u8> = serde_json::to_vec(&prev_data).unwrap();
+    let data = Vec::<u8>::new();
+    let wrong_call_results = Vec::<u32>::new();
+    let wrong_call_results = serde_json::to_vec(&wrong_call_results).unwrap();
+
+    let result = air::execute_air(
```

Comment:
It's better to run this through `vm.runner.call`, because it'll allow run it through Wasm.

Reply:
Left as is after the discussion.